src/SolutionTemplate/UnoSolutionTemplate/Shared/App.xaml.cs

```diff
@@ -23,7 +23,7 @@ namespace $ext_safeprojectname$
     /// </summary>
     public sealed partial class App : Application
     {
-        private Window _window;
+        internal static Window _window;
```

Comment:
I think the common convention for this is to be PascalCase?

Reply:
I have adjusted the code further to turn it into `MainWindow` property, which seemed more meaningful

Reply:
Yeah that is a lot better! 🎉